examples/multi_window.rs

```diff
@@ -0,0 +1,282 @@
+extern crate mini_gl_fb;
+extern crate glutin;
+
+use mini_gl_fb::glutin::event_loop::EventLoop;
+use mini_gl_fb::glutin::event::{Event, WindowEvent, MouseButton, VirtualKeyCode, KeyboardInput, ElementState};
+use mini_gl_fb::{get_fancy, GlutinBreakout, Config};
+use mini_gl_fb::glutin::dpi::{LogicalSize, LogicalPosition};
+use mini_gl_fb::glutin::window::{Window, WindowId, CursorIcon};
+use mini_gl_fb::glutin::event_loop::ControlFlow;
+use std::cell::Cell;
+use mini_gl_fb::glutin::platform::run_return::EventLoopExtRunReturn;
+use mini_gl_fb::glutin::{WindowedContext, PossiblyCurrent};
+
+/// Turn up this number to make the pixels bigger. 1 is one logical pixel
+const SCALE_FACTOR: f64 = 2.;
+
+/// A window being tracked by a `MultiWindow`. All tracked windows will be forwarded all events
+/// received on the `MultiWindow`'s event loop.
+trait TrackedWindow {
+    /// Handles one event from the event loop. Returns true if the window needs to be kept alive,
+    /// otherwise it will be closed. Window events should be checked to ensure that their ID is one
+    /// that the TrackedWindow is interested in.
+    fn handle_event(&mut self, event: &Event<()>) -> bool;
+}
+
+/// Manages multiple `TrackedWindow`s by forwarding events to them.
+struct MultiWindow {
+    windows: Vec<Cell<Box<dyn TrackedWindow>>>,
+}
+
+impl MultiWindow {
+    /// Creates a new `MultiWindow`.
+    pub fn new() -> Self {
+        MultiWindow {
+            windows: vec![],
+        }
+    }
+
+    /// Adds a new `TrackedWindow` to the `MultiWindow`.
+    pub fn add(&mut self, window: Box<dyn TrackedWindow>) {
+        self.windows.push(Cell::new(window))
+    }
+
+    /// Runs the event loop until all `TrackedWindow`s are closed.
+    pub fn run(&mut self, event_loop: &mut EventLoop<()>) {
+        if !self.windows.is_empty() {
+            event_loop.run_return(|event, _, flow| {
+                *flow = ControlFlow::Wait;
+
+                self.windows.retain(|window|
+                    unsafe { &mut *window.as_ptr() }.handle_event(&event)
```

Comment:
👀 but this is not in the core lib so /shrug if it runs 😄 - maybe at least comment it so nobody gets led astray

Reply:
Good now?